packages/zql/src/query/query.test.ts

```diff
@@ -856,16 +856,6 @@ test('custom materialize factory', () => {
   >();
 });
 
-test('Make sure that QueryInternal does not expose the ast', () => {
```

Comment:
API servers must pull the AST in order to send it back to Zero-cache.

Reply:
Can we make this an opaque type?